File: ts/management-api-v2/cm_api_v2_delete_taxonomy_group.ts
// Tip: Find more about JS/TS SDKs at https://kontent.ai/learn/javascript
import { ManagementClient } from '@kontent-ai/management-sdk';

const client = new ManagementClient({
  environmentId: 'KONTENT_AI_ENVIRONMENT_ID',
  apiKey: 'KONTENT_AI_MANAGEMENT_API_KEY'
});

const response = await client.deleteTaxonomy()
  .byTaxonomyId('dbff8416-c4c7-45d2-b497-a4a71a5cbe30')
  // .byTaxonomyCodename('personas_222')
  // .byTaxonomyExternalId('Tax-Group-124')
  .toPromise();

File: net/management-api-v2/cm_api_v2_delete_taxonomy_group.cs
// Tip: Find more about .NET SDKs at https://kontent.ai/learn/net
using Kontent.Ai.Management;

var client = new ManagementClient(new ManagementOptions
{
    ApiKey = "KONTENT_AI_MANAGEMENT_API_KEY",
    EnvironmentId = "KONTENT_AI_ENVIRONMENT_ID"
});


var identifier = Reference.ById(Guid.Parse("0be13600-e57c-577d-8108-c8d860330985"));
// var identifier = Reference.ByCodename("personas");
// var identifier = Reference.ByExternalId("Tax-Group-123");

await client.DeleteTaxonomyGroupAsync(identifier);
